rsmtool/notebooks/features_by_group.ipynb

```diff
@@ -38,7 +38,7 @@
     "    df_train_combined.reset_index(drop=True, inplace=True)\n",
     "\n",
     "    # decide on the the height per plot\n",
-    "    height_per_plot = 2 if len(features_used) > 15 else 4\n",
```

Comment:
The condition is in the function. In the old version of this notebook, `height_per_plot` is used to compute the total height of the figure by multiplying with the number of features. That is what [these](https://github.com/EducationalTestingService/rsmtool/blob/bugfix/plot-positions/rsmtool/utils.py#L423) [two](https://github.com/EducationalTestingService/rsmtool/blob/bugfix/plot-positions/rsmtool/utils.py#L428) lines do in the new function.